BlackNectar/Carthage/Checkouts/AromaSwiftClient/.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: objective-c
+cache: cocoapods
+rvm: 2.2.2
+osx_image: xcode8
+before_install:
+  - brew update
+
+  # Only if app has Carthage dependencies
+  - carthage bootstrap --verbose
```

Comment:
Great! By adding this, I believe you address issue #36 .
However, check the script and make sure that it works for this project.